pages/login/index.tsx

```diff
@@ -0,0 +1,98 @@
+import axios from 'axios';
+import Image from 'next/image';
+import Link from 'next/link';
+import { useRouter } from 'next/navigation';
+import { useEffect, useState } from 'react';
+
+import IconEyeInVisible from '@/public/images/icons/eye-invisible.svg';
+import IconEyeVisible from '@/public/images/icons/eye-visible.svg';
+import Logo from '@/public/images/logo/logo.svg';
+
+function LoginPage(): React.ReactElement {
+  const router = useRouter();
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [loginCheck, setLoginCheck] = useState(false); // 로그인 상태 체크
+
+  useEffect(() => {
+    const accessToken = sessionStorage.getItem('token');
+    if (accessToken) {
+      router.push('/');
+    }
+  }, [router]);
+
+  const handleLogin = async (e: React.MouseEvent<HTMLButtonElement>) => {
+    e.preventDefault();
+
+    try {
+      const res = await axios.post(
```

Comment:
P2:
응답값이 any로 추론되니 어떤 값이 올지 명시해주시면 더 좋을 것 같습니다

```suggestion
      const res = await axios.post<{accessToken: string; refreshToken: string;}>(
```